packit.yaml

```diff
@@ -1,12 +1,24 @@
 upstream_project_url: https://github.com/cockpit-project/cockpit-machines
 # enable notification of failed downstream jobs as issues
 issue_repository: https://github.com/cockpit-project/cockpit-machines
-specfile_path: cockpit-machines.spec
```

Comment:
the specfile_path was duplicated because apparently there is a bug in packit, when multiple `packages` are defined, they don't inherit the common fields. 

https://github.com/packit/packit/issues/2272